lib/src/results.dart

```diff
@@ -196,6 +197,9 @@ class RealmResults<T extends RealmObject> extends collection.IterableBase<T> {
   // void set length(int newLength) {
   //   throw new Exception("Setting length on Results<T> is not supported");
   // }
+
+  // TODO: For now changed don't look at the RealmCollectionChangesHandles
+  Stream<RealmResults> get changed => realmCore.resultChanged(this, _realm.scheduler.handle).map((_) => this);
```

Comment:
Should we call it `changes` instead? So user will write: `results.changes.listen(() { ... });` ?

Reply:
I think I prefer `changed` still.. Once we support full change sets, it will read like `results.changed.listen((changes) { ... });`